.github/workflows/release.yml

```
@@ -58,15 +58,17 @@ jobs:
         run: python -u ./dev/releases/make_archives.py
 
       - name: "Upload artifacts"
-        if: ${{ !startsWith(github.ref, 'refs/tags/v') && failure() }}
+        if: ${{ (!startsWith(github.ref, 'refs/tags/v') && failure()) || github.event_name == 'schedule' }}
```

```suggestion
        if: ${{ github.event_name == 'schedule' }}
```
Do we still want this condition any more?

---

It was added for debugging of failures of this step, so I'd say yes...

---

Cool, I'll keep it.